apps/public-docsite-v9/package.json

```diff
@@ -43,6 +43,7 @@
     "react": "17.0.2",
     "react-dom": "17.0.2",
     "react-window": "^1.8.6",
-    "tslib": "^2.1.0"
+    "tslib": "^2.1.0",
+    "@fluentui/react-rating-preview": "*"
```

Comment:
nit: Move this up with the other `@fluentui/react-` dependencies. (Ideally they'd all be sorted, but at least keeping them grouped is nice?)